tests/unit-tests/test-class-sensei-data-cleaner.php

```diff
@@ -77,6 +81,29 @@ private function setupPages() {
 		Sensei()->settings->get_settings();
 	}
 
+	/**
+	 * Add some users to run tests against. The roles and capabilities
+	 * associated with Sensei should be deleted on cleanup. The others should
+	 * not be deleted.
+	 */
+	private function setupUsers() {
+		// Create a regular user and assign some caps.
+		$this->regular_user_id = $this->factory->user->create( array( 'role' => 'author' ) );
+		$regular_user = get_user_by( 'id', $this->regular_user_id );
```

Comment:
You can use `create_and_get` when creating the user if you need the entire user object.

Reply:
I tried this, but it turns out that I have to reload the user objects in the test anyway, because the cleanup code changes them in the DB. Because of that, storing the ID in the instance var and loading the user when needed seems like the cleaner solution to me, rather than storing the entire object and having to reload it.

So I left this as-is, but feel free to weigh in if you disagree 😄